src/main/kotlin/upbrella/be/user/controller/UserController.kt

```
@@ -86,16 +88,52 @@ class UserController(
             ))
     }
 
+    @PostMapping("/users/oauth/apple/login")
+    fun appleLogin(session: HttpSession, @RequestBody code: LoginCodeRequest): ResponseEntity<CustomResponse<Unit>> {
+        val appleAccessToken: OauthToken
+
+        try {
+            appleAccessToken = oauthLoginService.getOauthToken(code.code, appleOauthInfo)!!
+        } catch (e: HttpClientErrorException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)